v2/init/dp-image-importer/service.patch

```diff
@@ -0,0 +1,29 @@
+diff --git service/service.go service/service.go
+index 3564d60..5e1434b 100644
+--- service/service.go
++++ service/service.go
+@@ -162,15 +162,15 @@ func registerCheckers(ctx context.Context,
+ 
+ 	hasErrors := false
+ 
+-	if err := hc.AddCheck("S3 private bucket", s3Private.Checker); err != nil {
+-		hasErrors = true
+-		log.Error(ctx, "error adding check for s3Private private bucket", err)
+-	}
+-
+-	if err := hc.AddCheck("S3 uploaded bucket", s3Uploaded.Checker); err != nil {
+-		hasErrors = true
+-		log.Error(ctx, "error adding check for s3Private uploaded bucket", err)
+-	}
++	// if err := hc.AddCheck("S3 private bucket", s3Private.Checker); err != nil {
++	// hasErrors = true
++	// log.Error(ctx, "error adding check for s3Private private bucket", err)
++	// }
++
++	// if err := hc.AddCheck("S3 uploaded bucket", s3Uploaded.Checker); err != nil {
++	// hasErrors = true
++	// log.Error(ctx, "error adding check for s3Private uploaded bucket", err)
++	// }
```

Comment:
There's commented out code, which needs to be removed I think.

Reply:
I've now edited that code to show that the commented-out code is a temporary fix (so that the app will run healthily in dp-compose)
but that the changes should not be committed - it's a workaround for dp-compose which may be resolved later

Reply:
That makes sense, thanks, I'll approve.